filestack/exceptions.py

```diff
@@ -0,0 +1,7 @@
+class FilestackHTTPError(Exception):
+    """
+    Custom HTTPError instead of requests.exceptions.HTTPError to add response body.
+
+    References:
+        - https://github.com/psf/requests/pull/4234
+    """
```

Comment:
Moved this to a new module, since i assume people would want to import this as 
`from filestack.exceptions import FilestackHTTPError` similar to requests.